scripts/commitcheck.sh

```diff
@@ -16,7 +16,7 @@ function test_url()
 # check for a tagged line
 function check_tagged_line()
 {
-    regex='^\s*'"$1"':\s+\S+\s+\<\S+\>'
+    regex='^\s*'"$1"':\s[-A-Za-z0-9 ]+\s<\S+>$'
```

Comment:
This regex doesn't look broad enough.  I'd suggest running it over the email addresses from the last couple hundred commits and makes sure it works.  For example, it needs to contain `.`.